benchmark/run.js

```diff
@@ -36,7 +36,8 @@ if (format === 'csv') {
 
 (function recursive(i) {
   const filename = benchmarks[i];
-  const child = fork(path.resolve(__dirname, filename), cli.optional.set);
+  const child = fork(path.resolve(__dirname, filename), cli.optional.set,
+                     { execArgv: ['--expose_internals'] });
```

Comment:
It is for people that try `node benchmark\misc\freelist.js` and encounter problems.